auth/jwt.go

```diff
@@ -72,7 +92,20 @@ func (s *JWTSecret) Validate(tokenString string) (jwt.MapClaims, error) {
 
 // SetSecret allows for the secret of the signer to be set, but not exposed.
 func (s *JWTSecret) SetSecret(secret []byte) {
```

Comment:
this has a failure case that leaves the user untouched as far as I can tell, but as a caller you don't know whether it actually editted the JWTSecret or not. I don't know if returning an error or something like that is overkill but since it's a public method I'd expect some feedback about the failure

Reply:
Even just some doc that you should call ValidSecret first and not ignore its result

Reply:
Good catch, definitely could be buggy.